src/composables/transactionTx.ts

```diff
@@ -89,6 +90,11 @@ export function useTransactionTx({
 
     return !!(
       innerTx.value?.contractId
+      && innerTx.value?.function
+      && (
+        Object.values(flatten(Object.values(FUNCTION_TYPE_DEX)))
+          .includes(innerTx.value?.function as TxFunctionRaw)
+      )
```

Comment:
I don't see the need for the first `Object.values()` call. Also we could reduce our dependency on external libraries and use plain js (but that's just a suggestion).
```suggestion
      && (
        Object.values(FUNCTION_TYPE_DEX).flat()
          .includes(innerTx.value?.function as TxFunctionRaw)
      )
```